getting-started/getting-help.rst

```diff
@@ -13,40 +13,53 @@ Should you require help, there are a :ref:`variety of options available
 usage then please check the rest of this guide first as it should answer your
 question.
 
+
+.. _help-discourse:
+
 Discourse
 ---------
 
-Python has a hosted :ref:`Discourse <discourse_discuss>` instance. This forum has many different
-categories and most core development discussions take place in the open forum
-categories for `PEPs`_ and `Core Development`_ .
-Most categories are open for all users to read and post with the exception of
-Committers and Core Development categories. Be sure to visit the related Core
-categories, such as
-`Core Development <https://discuss.python.org/c/core-dev/23>`_ and
-`Core Workflow <https://discuss.python.org/c/core-workflow/8>`_.
+Python has a hosted Discourse instance at `discuss.python.org <Discourse_>`_.
+This forum has many different categories,
+of which all but `Committers`_ are open for all users to read and post.
+For most general questions and help related to the development
+of the Python language, interpreter and standard library itself
+(as opposed to developing your own libraries and applications *with* Python),
+the `Core Development category`_ is the place to ask.
+The `Ideas category`_ is the place to propose and discuss new ideas
+for the Python language and standard library,
+while the `Core Workflow category`_ is the place to post
+about questions, concerns and issues with the dev infrastructure and bots.
```

Comment:
Maybe this would work better as a list?
```rst
* `Core Development`_: for general questions...
* `Ideas`_: for proposing and discussing new ideas...
* `Core Workflow`_: for questions, concerns, and issues with...
```

Reply:
Sure, implemented as suggested, with more concise descriptions as well